Mathlib/CategoryTheory/Limits/Preserves/FunctorCategory.lean

```diff
@@ -71,15 +73,25 @@ def FunctorCategory.prodPreservesColimits [HasBinaryProducts D] [HasColimits D]
             · intro G G'
               apply prodComparison_natural ((evaluation C D).obj k) (𝟙 F) } ) }
 
-instance whiskeringLeftPreservesLimits [HasLimits D] (F : C ⥤ E) :
+end
+
+variable {C : Type u₁} [Category.{v₁} C]
+variable {D : Type u₂} [Category.{v₂} D]
+variable {E : Type u₃} [Category.{v₃} E]
+
+instance whiskeringLeftPreservesLimitsOfShape (J : Type u) [Category.{v} J]
+    [HasLimitsOfShape J D] (F : C ⥤ E) :
+    PreservesLimitsOfShape J ((whiskeringLeft C E D).obj F) :=
+  ⟨fun {K} =>
+    ⟨fun c {hc} => by
+      apply evaluationJointlyReflectsLimits
+      intro Y
+      change IsLimit (((evaluation E D).obj (F.obj Y)).mapCone c)
+      exact PreservesLimit.preserves hc⟩⟩
+
+instance whiskeringLeftPreservesLimits [HasLimitsOfSize.{(max u₁ v₂)} D] (F : C ⥤ E) :
```

Comment:
Could you add a more general `PreservesLimitsOfSize` instance, with arbitrary universe parameters and suitable `HasLimitsOfSize` assumption?

Reply:
We don't need the more specific one then, do we? Mirrored the way `whiskeringRightPreservesLimits` does it.